hadoop/pom.xml

```diff
@@ -34,7 +34,7 @@
 
     <properties>
         <version.org.infinispan>8.5.0-redhat-SNAPSHOT</version.org.infinispan>
-        <version.jdg.hadoop>0.2.0.Final-redhat-2</version.jdg.hadoop>
+        <version.jdg.hadoop>0.3.0-redhat-SNAPSHOT</version.jdg.hadoop>
```

Comment:
Did't we agree to use only released versions?

//cc @tristantarrant @gustavonalle

Reply:
We decided to start building hadoop again because of JDG-1189, which requires update with each JDG build